public/app/features/plugins/built_in_plugins.ts

```diff
@@ -54,8 +52,6 @@ import * as barGaugePanel from 'app/plugins/panel/bargauge/module';
 import * as logsPanel from 'app/plugins/panel/logs/module';
 import * as newsPanel from 'app/plugins/panel/news/module';
 
-const exampleApp = async () => await import(/* webpackChunkName: "exampleApp" */ 'app/plugins/app/example-app/module');
```

Comment:
Deleting the app and we can maintain it from:
https://github.com/grafana/simple-app-plugin